boards/nucleo144-f207/include/periph_conf.h

```diff
@@ -258,16 +258,18 @@ static const spi_conf_t spi_config[] = {
 
 /**
  * @name    ADC configuration
- *
- * We need to define the following fields:
- * PIN, device (ADCx), channel
  * @{
  */
-#define ADC_CONFIG {              \
-    {GPIO_PIN(PORT_A, 3), 0, 3},  \
-    {GPIO_PIN(PORT_C, 0), 1, 0}  \
-}
-#define ADC_NUMOF          (2)
+static const adc_conf_t adc_config[] = {
+    { .pin = GPIO_PIN(PORT_A,  3), .dev = ADC_1, .chan =  3 },
+    { .pin = GPIO_PIN(PORT_C,  0), .dev = ADC_1, .chan = 10 },
+    { .pin = GPIO_PIN(PORT_C,  3), .dev = ADC_1, .chan = 13 },
+    { .pin = GPIO_PIN(PORT_F,  3), .dev = ADC_3, .chan =  9 },
```

Comment:
The build fails for me because of the use of ADC_3. Even if it's compliant with the board datasheet.